WebContent/js/pvws.js

```diff
@@ -78,11 +78,19 @@ class PVWS
             if (value === undefined)
                 value = { pv: jm.pv, readonly: true };
             
-            // Update cached value with received changes
-            Object.assign(value, jm);
-            this.values[jm.pv] = value;
-            // console.log("Update for PV " + jm.pv + ": " + JSON.stringify(value));
-            this.message_handler(value);
+            //Only way found to show PV as disconnected
+            if(jm.value =="NaN")
```

Comment:
Had to revert this.
A value may well be 'NaN'. Having one PV with that value must not result in closing the web socket.
The web socket is shared by many PVs, so with this change one NaN PV will disconnect all PVs, then reconnect, again get NaN, ... a never ending cycle.